README.md

```diff
@@ -1 +1,70 @@
-# businessCardReact
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
Warto to uzupełnić czymś od siebie - np po co zrobiłeś to zadanie, kto je sprawdzał itd. Przydatne z punktu widzenia rektureta